src/Admin/UrlGeneratorInterface.php

```diff
@@ -42,6 +42,7 @@ public function setRouteGenerator(RouteGeneratorInterface $routeGenerator);
      *
      * @param string $name
      * @param mixed  $object
+     * @param array  $parameters
```

Comment:
I think not including that one was deliberate, since it gives no additional information

Reply:
Same, no point adding it if there is no comment about the field

Reply:
IMO we should remove it completely or if we keep the php doc it should be complete.
The other have no additional information, too...

Reply:
There is no `mixed` type and also we support php 5.6 so we can't have a `string` type. I don't know, this is more of a developers POV, maybe we should define a standard and use it everywhere? In my case, I use partial comments (eg comment for types you can't do in your code)

Reply:
@OskarStark @kunicmarko20 please read https://github.com/sonata-project/dev-kit/issues/345

Reply:
then we can close this?

Reply:
I think so